packages/atomic/src/components/atomic-result/atomic-result.tsx

```diff
@@ -50,6 +104,11 @@ export class AtomicResult {
   }
 
   public render() {
-    return <div innerHTML={this.content}></div>;
+    return (
+      <div
+        class={`result-root ${this.getDisplayClass()} ${this.getDensityClass()} ${this.getImageClass()}`}
+        innerHTML={this.content}
```

Comment:
Is there any chance of XSS making it's way into `content`?

Reply:
Currently `content` is given whatever the content of a `template` element is, so yes. However, in this particular use case it's not very relevant since if somebody can add a script in the template they would've been capable of adding it anywhere anyways.

If somebody decides to create a `atomic-result` element without the result list, then again `content` can contain an XSS, so they would have to do their own sanitization.

Reply:
Currently `content` is given whatever the content of a `template` element is, so yes. However, in this particular use case it's not very relevant since if somebody can add a script in the template they would've been capable of adding it anywhere anyways.

If somebody decides to create a `atomic-result` element without the result list, then again `content` can contain an XSS, so they would have to do their own sanitization.

Reply:
Ok, if it's only the dev building the page who can inject the XSS, then all is good. The important thing is to prevent someone from using the searchapi in conjunction with this line as a vector to inject XSS,